3D game demo/COMP220/06_Lighting/Mesh.cpp

```diff
@@ -1,14 +1,14 @@
 #include "stdafx.h"
-#include "Mesh.h"
+#include "mesh.h"
```

Comment:
class name should have a capital letter so it's easier to identify that it's a class.